portal-api/src/v1/gef/routes.js

```diff
@@ -1,31 +1,50 @@
 const express = require('express');
 const { validate } = require('../../role-validator');
-const mandatoryCriteriaVersioned = require('./controllers/mandatoryCriteriaVersioned.controller');
+
 const application = require('./controllers/application.controller');
+const exporter = require('./controllers/exporter.controller');
+const facilities = require('./controllers/facilities.controller');
+const mandatoryCriteriaVersioned = require('./controllers/mandatoryCriteriaVersioned.controller');
 
 const router = express.Router();
 
 // Application
 router.route('/application')
-  .get(validate({ role: ['maker', 'checker'] }), application.getAll)
-  .post(validate({ role: ['maker'] }), application.create);
+  .get(validate({ role: ['checker', 'data-admin'] }), application.getAll)
```

Comment:
should this also include 'maker'? As this stands, a maker cannot list all applications. Is this whats required?

Reply:
updated